.idea/compiler.xml

```diff
@@ -15,7 +15,7 @@
   </component>
   <component name="JavacSettings">
     <option name="ADDITIONAL_OPTIONS_OVERRIDE">
-      <module name="BookStore" options="-parameters" />
+      <module name="BookStore" options="" />
```

Comment:
What was the purpose of this change? Curious whether this is an environmentally sensitive value?

Reply:
Not sure why but IntelliJ made this change automatically. I think we should probably leave it anyways.